core/src/scene/sceneLoader.cpp

```diff
@@ -876,10 +876,7 @@ Filter SceneLoader::generateNoneFilter(Node _filter, Scene& scene) {
             filters.emplace_back(generateFilter(filt, scene));
         }
     } else if (_filter.IsMap()) { // not case
-        for (const auto& filt : _filter) {
-            std::string keyFilter = filt.first.as<std::string>();
-            filters.emplace_back(generatePredicate(_filter[keyFilter], keyFilter));
-        }
+        filters.emplace_back(generateFilter(_filter, scene));
     } else {
         LOGW("Invalid filter. 'None' expects a list or an object.");
         return Filter();
```

Comment:
Ah yeah, whoops. That's exactly the thing I was talking about in an earlier comment haha. It should be a simple matter of replacing `_filter.IsMap()` with `_filter.IsMap() || _filter.IsScalar()`, then `generateFilter` will take care of detecting a function in a scalar. I'll push that fix to master if nobody is opposed.